extensions/amp-subscriptions/0.1/local-subscription-platform-base.js

```diff
@@ -109,27 +109,24 @@ export class LocalSubscriptionBasePlatform {
    * @protected
    */
   initializeListeners_() {
-    const handleClickAndStopEventPropagation = e => {
-      e.stopPropagation();
+    const handleClickOncePerEvent = e => {
+      if (e.ampSubscriptionsClickHasBeenHandled) {
```

Comment:
As a surrogate property we need to access it via `[]`. E.g. `e['_AMP_SUBS_CLICK_HANDLED']` or something like this. Would probably be best to put it in a const.

Reply:
Good call :+1: